### DISCUSSION TITLE
cover notifications in fullscreen mode

### DISCUSSION THREAD

**Discussion by karb94** [hyprwm/Hyprland#7454] [Q&A] [answered]
Is there a way to cover/hide notify-send type of notifications when making a window full screen?

**Comment by vaxerski** [comment, marked as answer]
have them on the top layer and not overlay

**Comment by karb94** [comment]
Thank you! For anyone reading this post, you configure the layer in your notification manager config not on Hyprland. In dunst you can set `layer = top` in your `dunstrc`
